src/Utils/errorHandler.ts

```diff
@@ -0,0 +1,10 @@
+class ErrorHandler extends Error {
```

Comment:
There are two classes with the same name, check `error-handler.ts` and `errorHandler.ts`, delete the one you aren't using.

Reply:
WIP